world/entity/entitytypes.py

```diff
@@ -26,6 +26,14 @@ def getEntity(iden):
 	def load():
 		for ent in EntityTypes:
 			if ent.value.texturePath is not None:
-				ent.value.image = pygame.transform.scale(
-					Assets[ent.value.texturePath.upper()].value.image, (64, 64))
-				ent.value.rect = ent.value.image.get_rect()
+				try:
+					ent.value.image = pygame.transform.scale(
+						Assets[ent.value.texturePath.upper()].value.image, (64, 64))
+					ent.value.rect = ent.value.image.get_rect()
+				except KeyError:
```

Comment:
Dit is de hele rede dat je dit niet wilt doen. Dit is puur nog de initialization die nu 2x zo lang duurt voor custom assets want python exception handling

Reply:
done